src/System.Net.Sockets/src/System/Net/Sockets/DynamicWinsockMethods.cs

```diff
@@ -261,6 +261,7 @@ private void EnsureTransmitPackets(SafeCloseSocket socketHandle)
         }
     }
 
+    [UnmanagedFunctionPointerAttribute(CallingConvention.StdCall, SetLastError=true)]
```

Comment:
Nit: Could be simplified to `[UnmanagedFunctionPointer(CallingConvention.StdCall, SetLastError=true)]` (without the `Attribute` suffix).